### PR TITLE
block concept changes

### PR DESCRIPTION
**Pull request by sunnya97**
More precision to labels of Block Data Properties

### CODE REVIEW

**Comment by webthethird** [comment]
Solid improvements to the block concept definitions, nothing that strikes me as a major change

File: EthOn.rdf [hunk 1]
```diff
@@ -914,6 +916,7 @@
 
     <owl:DatatypeProperty rdf:about="http://ethon.consensys.net/blockGasLimit">
         <rdfs:subPropertyOf rdf:resource="http://ethon.consensys.net/BlockDataProperty"/>
+        <rdfs:subPropertyOf rdf:resource="http://ethon.consensys.net/NetworkDataProperty"/>
```
Comment (jo-tud) [inline EthOn.rdf]: This line should be deleted, I think (926)

Comment (webthethird) [inline EthOn.rdf]: I just deleted it

Comment (webthethird) [inline EthOn.rdf]: Does everything else here look good?

**Pull request merged**